GenderPayGap.WebUI/Repositories/UserRepository.cs

```diff
@@ -190,28 +188,8 @@ private void UpdateUserPasswordUsingPBKDF2(User user, string password)
             user.PasswordHash = Crypto.GetPBKDF2(password, salt);
             user.HashingAlgorithm = HashingAlgorithm.PBKDF2;
 
-            dataRepository.SaveChangesAsync().Wait();
+            dataRepository.SaveChanges();
         }
 
-        #region IDataTransaction
-
-        public async Task BeginTransactionAsync(Func<Task> delegateAction)
-        {
-            await dataRepository.BeginTransactionAsync(delegateAction);
-        }
-
-        public void CommitTransaction()
-        {
-            dataRepository.CommitTransaction();
-        }
-
-        public void RollbackTransaction()
-        {
-            dataRepository.RollbackTransaction();
-        }
-
-        #endregion
-

```

Comment:
This was never being used.
`UserRepository` implements `IUserRepository` which implented `IDataTransaction` which has these methods on it - but no-one ever used a `UserRepository` transactionally